src/utils.ts

```diff
@@ -105,8 +106,33 @@ function readCompilerOptions(configPath: string) {
   return parsedConfig.options;
 }
 
+export function getTSConfigOptionFromConfig(globals: any) {
+  if (!globals) { return 'tsconfig.json'; }
+
+  const tsJestConfig = getTSJestConfig(globals);
+
+  if (('__TS_CONFIG__' in globals) && globals.__TS_CONFIG__) {
```

Comment:
Do we need the double check here? Shouldn't globals.__TS_CONFIG__ simply return false if it's undefined or not set (as the in operator does)